project/models.py

```diff
@@ -66,16 +78,53 @@ class Package(models.Model):
     created = models.DateTimeField(auto_now_add=True, db_index=True)
     updated = models.DateTimeField(auto_now=True, db_index=True, null=True)
 
-    # Additional permissions to manage members.
-    # (Add, change, delete, and view permissions are automatically created)
     class Meta:
         permissions = [
-            ("can_add_members", "Can add members"),
-            ("can_delete_members", "Can delete members"),
+            manage_project_users_permission,
+            edit_project_permission,
+            view_project_permission,
         ]
 
     def __str__(self):
         return "%s id=%d" % (self.title, self.id)
 
     def get_absolute_url(self):
         return f"/packages/{self.id}"
+
+
+@receiver(post_save, sender=Package)
+def create_groups_for_project(sender, instance, **kwargs):
+    print("*----- create_groups_for_project function -----*")
+    pprint("---sender")
+    pprint(sender)
+    pprint("---instance")
+    pprint(instance)
+    pprint("---kwargs")
+    pprint(kwargs)
+
+    # only want to do this when a project is created
```

Comment:
Is `created` a boolean value that always exists in the kwargs? If not you may want to use `if "created" in kwargs:`

Reply:
Ah I like that suggestion, but it does show up as a key with a boolean value. Here's the docs for the post_save https://docs.djangoproject.com/en/3.2/ref/signals/#post-save

This is an example of what we get:
```
{'created': True,
 'raw': False,
 'signal': <django.db.models.signals.ModelSignal object at 0x7fdead30f430>,
 'update_fields': None,
 'using': 'default'}
```